### ISSUE TITLE
Vegetable Garden Project: duplicate xml node

### ISSUE THREAD

**Issue by SaucyPigeon** [SaucyPigeon/Wild-Cultivation-Fan-Update#7]
troopersmith1 11 hours ago 
With both this and VGP loaded I'm getting a red error. Something about duplicate xml node for sowtags, load order doesn't matter.

**Comment by SaucyPigeon** [comment]
```
XML error: Duplicate XML node name sowTags in this XML block: <plant><fertilitySensitivity>0.7</fertilitySensitivity><growDays>3</growDays><harvestWork>60</harvestWork><maxMeshCount>9</maxMeshCount><visualSizeRange>0.7~0.85</visualSizeRange><topWindExposure>0.4</topWindExposure><wildClusterRadius>4</wildClusterRadius><wildClusterWeight>10</wildClusterWeight><wildOrder>1</wildOrder><sowTags><li>Ground</li><li>Ground</li></sowTags><sowTags><li>Ground</li></sowTags></plant>

Root node: <ThingDef ParentName="PlantBase"><defName>Plant_TallGrass</defName><label>tall grass</label><description>Wild tall grass. Slows down anyone moving over it.</description><statBases><MaxHitPoints>90</MaxHitPoints><Flammability>1.3</Flammability><Nutrition>0.30</Nutrition></statBases><pathCost>15</pathCost><graphicData><texPath>Things/Plant/Grass</texPath><graphicClass>Graphic_Random</graphicClass></graphicData><hideAtSnowDepth>0.5</hideAtSnowDepth><ingestible /><plant><fertilitySensitivity>0.7</fertilitySensitivity><growDays>3</growDays><harvestWork>60</harvestWork><maxMeshCount>9</maxMeshCount><visualSizeRange>0.7~0.85</visualSizeRange><topWindExposure>0.4</topWindExposure><wildClusterRadius>4</wildClusterRadius><wildClusterWeight>10</wildClusterWeight><wildOrder>1</wildOrder><sowTags><li>Ground</li><li>Ground</li></sowTags><sowTags><li>Ground</li></sowTags></plant></ThingDef>
Verse.Log:Error(String, Boolean)
Verse.XmlInheritance:CheckForDuplicateNodes(XmlNode, XmlNode)
Verse.XmlInheritance:CheckForDuplicateNodes(XmlNode, XmlNode)
Verse.XmlInheritance:ResolveXmlNodeFor(XmlInheritanceNode)
Verse.XmlInheritance:ResolveXmlNodesRecursively(XmlInheritanceNode)
Verse.XmlInheritance:ResolveXmlNodesRecursively(XmlInheritanceNode)
Verse.XmlInheritance:ResolveXmlNodes()
Verse.XmlInheritance:Resolve()
Verse.LoadedModManager:ParseAndProcessXML(XmlDocument, Dictionary`2)
Verse.LoadedModManager:LoadAllActiveMods()
Verse.PlayDataLoader:DoPlayLoad()
Verse.PlayDataLoader:LoadAllPlayData(Boolean)
Verse.<>c:<Start>b__6_1()
Verse.LongEventHandler:RunEventFromAnotherThread(Action)
Verse.<>c:<UpdateCurrentAsynchronousEvent>b__27_0()
System.Threading.ThreadHelper:ThreadStart_Context(Object)
System.Threading.ExecutionContext:RunInternal(ExecutionContext, ContextCallback, Object, Boolean)
System.Threading.ExecutionContext:Run(ExecutionContext, ContextCallback, Object, Boolean)
System.Threading.ExecutionContext:Run(ExecutionContext, ContextCallback, Object)
System.Threading.ThreadHelper:ThreadStart()
```

**Comment by SaucyPigeon** [comment]
Duplicate XML node caused by both VGP and WC making tall grass cultivable. Other plants that are made cultivable by both mods include normal grass and pincushion cactus.

**Comment by SaucyPigeon** [comment]
Added patch for VGP to avoid duplicate plants. Fixed in version 1.0.5.
